Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -300,6 +300,19 @@
     items:
     - RCDRecharging
 
+- type: entity
+  id: BorgModuleRPD
+  parent: [ BaseBorgModuleEngineering, BaseProviderBorgModule ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)